#!/usr/bin/env python3


PACKAGE = 'dutuuv_control'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('angular_z_kp', double_t, 0, "angular z axis velocity", default=1.0, min=0.1, max=3.0)
gen.add('angular_z_ki', double_t, 0, "angular z axis velocity", default=0.01, min=0.01, max=1.0)


exit(gen.generate(PACKAGE, 'wrench_controller', 'WrenchControllerPid'))